build/vs2022/box16.vcxproj

```diff
@@ -267,6 +267,7 @@ xcopy $(VendorDir)\zlib-1.2.13\lib\x86\*.dll "$(OutDir)" /E /I /F /Y
       <WarningLevel>Level3</WarningLevel>
       <TreatWarningAsError>true</TreatWarningAsError>
       <PreprocessorDefinitions>ZLIB_WINAPI;</PreprocessorDefinitions>
+      <DisableLanguageExtensions>false</DisableLanguageExtensions>
```

Comment:
Is this change (and the one below in this file) necessary? I'm guessing not since the win32 configurations appear to build and the x64 release-trace configuration appears to build. If you want, I can edit this file manually to remove these added lines.